pallets/transaction-multi-payment/src/weights.rs

```diff
@@ -0,0 +1,82 @@
+// This file is part of hack.HydraDX-node.
+
+// Copyright (C) 2021 Parity Technologies (UK) Ltd.
```

Comment:
Can we regen this to intergalactic or public domain or something? 😄

Reply:
Oh..i did not notice..  sure.

Reply:
done.